cmd/utils.go

```diff
@@ -163,20 +164,25 @@ var globalProfiler interface {
 func dumpRequest(r *http.Request) string {
 	header := cloneHeader(r.Header)
 	header.Set("Host", r.Host)
+	// Replace all '%' to '%%' so that printer format parser
+	// to ignore URL encoded values.
+	rawQuery := strings.Replace(r.URL.RawQuery, "%", "%%", -1)
+	rawPath := strings.Replace(getURLEncodedName(r.URL.Path), "%", "%%", -1)
 	req := struct {
 		Method string      `json:"method"`
 		Path   string      `json:"path"`
 		Query  string      `json:"query"`
 		Header http.Header `json:"header"`
-	}{r.Method, getURLEncodedName(r.URL.Path), r.URL.RawQuery, header}
-	jsonBytes, err := json.Marshal(&req)
-	if err != nil {
+	}{r.Method, rawPath, rawQuery, header}
```

Comment:
```
	req := struct {
		Method string      `json:"method"`
		URI    string      `json:"uri"`
		Query  string      `json:"query"`
		Header http.Header `json:"header"`
	}{r.Method, r.RequestURI, header}
```

this would work better